src/styles/blocks/card.scss

```diff
@@ -0,0 +1,106 @@
+.card {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(1, 200px);
+  column-gap: 48px;
+  row-gap: 46px;
+  padding: 50px 15px;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+
+  &__item {
+    box-sizing: border-box;
+    width: 200px;
+    border-radius: 5px;
+    border: 1px solid #f3f3f3;
+    background: #fff;
+    padding: 0 16px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    transition: all 0.3s ease 0s;
+    &:hover {
+      transform: scale(1.2);
+    }
+  }
+
+  &__image {
+    padding-top: 32px;
+    margin-bottom: 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: #060b35;
+    margin-bottom: 4px;
+    .card__item:hover .card__title {
+      color: #34568b;
+    }
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__stars {
+    margin-bottom: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__cost {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__name {
+    font-size: 12px;
```

Comment:
you don't need to duplicate font-size since you've applied it to the entire document. you just need to override it where it is different.
please check your styles for these duplicates and fix them